packages/terra-switch/src/Switch.module.scss

```diff
@@ -0,0 +1,199 @@
+// Themes
+@import './clinical-lowlight-theme/Switch.module';
+@import './orion-fusion-theme/Switch.module';
+
+:local {
+  .switch {
+    background-color: transparent;
+    border: 0;
+    display: flex;
```

Comment:
Update to `display: inline-flex`.

Reply:
updated to [inline-flex](https://github.com/cerner/terra-core/pull/2846/commits/893ebe62f9fd9578c95be4985af8c5a9b9545525#diff-323c0506d3b9781e723ca3d110a89c79)